staging/src/k8s.io/apimachinery/pkg/selection/operator.go

```diff
@@ -31,3 +36,25 @@ const (
 	GreaterThan  Operator = "gt"
 	LessThan     Operator = "lt"
 )
+
+var (
+	OperatorMap map[string]Operator = map[string]Operator{
```

Comment:
Prefer to leave this file untouched.

We don't need to replace every switch by maps. That's not necessarily going to give a performance boost (although you could try and show results from experiments).